magento/inventory
# Integrate the Sales Channel output to the Manage Stock view

### Description
Integrate the Sales Channel output to the Manage Stocks view and remove previous used dummy data with the data provided by the database.

### Manual testing scenarios
1. Go to the _"Manage Stocks"_ view in **Stores** -> **Settings**-> **Manage Stocks** 
2. Check if the correct sales channels are displayed in the grid

### Contribution checklist
